### PR TITLE
resource header + navigation for s1

### PR DESCRIPTION
**Pull request by der-john**
continues #2549. needs #2573 - that's why it's 'on hold'.

(replaces #2560.)

### CODE REVIEW

File: src/s1/s1/static/js/Packages/S1/Proposal/Detail.html [hunk 1]
```diff
@@ -1,5 +1,16 @@
...
     <header class="resource-header" data-ng-class="{'m-image': data.picture}">
+        <div class="resource-header-navigation">
+            <adh-resource-dropdown
+                data-resource-path="{{path | adhParentPath}}"
+                data-ng-if="path"
+                data-edit="true"
+                data-report="true"
+                data-hide="true"
+                data-print="true"
+                data-cancel="true">
```
Comment (xi) [inline src/s1/s1/static/js/Packages/S1/Proposal/Detail.html]: cancel?

**Comment by xi** [comment]
I did some cleanups. Feel free to merge.

File: src/s1/s1/static/js/Packages/S1/Proposal/Detail.html [hunk 2]
```diff
@@ -1,4 +1,4 @@
-<div class="s1-proposal-detail is-{{data.workflowState}}" data-ng-if="data.title">
+<div data-ng-if="data.title">
```
Comment (der-john) [inline src/s1/s1/static/js/Packages/S1/Proposal/Detail.html]: nice.

**Comment by der-john** [comment]
looks good & works.

**Pull request merged**